entrypoint.sh

```diff
@@ -48,13 +48,24 @@ fi
 
 echo $log
 
+# this will bump the semvar using the default bump level,
+# or it will simply pass if the default was "none"
+function default-bump {
+  if [ "$default_semvar_bump" == "none" ]; then
+    echo "Default bump was set to none. Skipping..."
+    return
```

Comment:
Should this `exit 0` here? Will a return quit the script early or just return from the function? (I'm not much of a bash expert)

Reply:
I suppose we could do `exit 0` here. It comes down to if you want the rest of the script to run if the bump was disabled.

Looking through that, it seems like we probably _don't_ want the rest of the script to run, so an exit will work